src/sass/form.scss

```diff
@@ -0,0 +1,29 @@
+.iti-flag {
+    background-image: url("/img/flags.png");
```

Comment:
use relative path `./img/...`

Reply:
with relative path doesn't work, used this way to reflect url path